pennylane/optimize/gradient_descent.py

```diff
@@ -63,8 +63,8 @@ def step_and_cost(self, objective_fn, *args, grad_fn=None, **kwargs):
 
         Returns:
             tuple[list [array], float]: the new variable values :math:`x^{(t+1)}` and the objective
-                function output prior to the step.
-                If single arg is provided, list [array] is replaced by array.
```

Comment:
Continuation lines in `Returns:` should never be indented; if they are, sphinx will treat them as a glossary entry and bold the first line.